README.md

```diff
@@ -1 +1,48 @@
-# camera_fusion
\ No newline at end of file
+# Camera Fusion Package
+
+This repository contains a C++ package for fusing point clouds from two different cameras using ROS 2. The package also supports 2D images. It is designed to work with RealSense 2 and Azure Kinect cameras.
+
+# Prerequisites
+
+Before getting started, make sure you have the following:
+
+1. ROS 2 (compatible version with your dependencies)
+2. Drivers for RealSense 2 and Azure Kinect cameras
+3. Colcon for building ROS 2 packages
+
+# Installation and Usage Steps
```

Comment:
Might want to specify that the intended usage is for the `curobo_ros` project and what setup (if any) it requires in that context.

Reply:
That is true but the code is not reliate to curobo function so it technicaly work for anything reliate to ROS2 and fusioning 2 pointcloud of camera sending depth image. So the context should be just after a calibration of two camera ?

Reply:
That is true, but specifying your intent can justify certain design decisions or restrictions. For example, the specific camera combination currently supported.